### ISSUE TITLE
Ability: Echoing Reprimand (Rogue - Kyrian - Covenant Legendary: Resounding Clarity) [buff-stack tracking does not work]

### ISSUE THREAD

**Issue by anianiani123** [ascott18/TellMeWhen#1922]
Hi.

TLDR:
tellmewhen does only track the highest Echoing Reprimand number-buff; starting with number 5. And only starts tracking lower numbers, when higher numbers have been used.

---

I am trying to track the 4 Echoing Reprimand buffs, but tellmewhen doesn't do it, unlike weakauras, where it works, but I prefer tellmewhen, so I want it to work there.

When you use Echoing Reprimand, with the Kyrian Covenant Legendary, you get 4 buffs, all with the same name (Echoing Reprimand), but these 4 buffs all have a different "stack-number" attached to it. Number 2, 3, 4, and 5.
When you use a two combo-point finisher, Echoing Reprimand with the number 2 gets used and disappears.

(tellmewhen icon)
If I am trying to track Echoing Reprimand with 2 stacks, for my 2 combo point finisher (with "Condition" "Buff - Stacks", 2 stacks of Echoing Reprimand, in combination with 2 active combo-points), tellmewhen only starts tracking the 2 stack combo-point finisher when number 5, 4, and 3 have been used. Tellmewhen only reacts to the highest combo-point finisher buff currently active. When Echoing Reprimand number 5 has been used, tellmewhen will start tracking number 4, then number 3; and then when I have used a 5, 4, and 3 combo-point finisher, it will start tracking what I actually wanted to track, the number 2 Echoing Reprimand, to alert me when I have two combo points active, and a 2 stack Echoing Reprimand active, so I can cast a Finishing move.

The tellmewhen icon for 2 stacks only appears ("starts tracking) when number 5, 4, and 3 have been used.

Not sure if this is enough to understand the problem. And sorry for the messy explanation of the problem.

(when I track Echoing Reprimand with 2 stacks on an weakaura icon, it just works. 3 stacks works too. etc)

I  also tried the "Condition" "Buff - Aura Variable", even though I don't quite understand what it does. Doesn't work.

--

I might just be dumb, and this is a dumb question. If so, tell me >.>

Greetings from Austria

Alexander

**Comment by anianiani123** [comment]
If I track Echoing Reprimand with 5 stacks and 5 combo points, it instantly works, because it's the highest number.

**Comment by ascott18** [comment]
Please do not ignore the issue template that is provided when you create a new issue. The questions and sections within it provide me with essential information to be able to diagnose and resolve your issue. If you do not include this information, there is little I can do to help you.

---

Please post the export string(s) of the icons, groups, or profiles you're having trouble with. 

To get an export string, open the icon editor, and click the button labeled "Import/Export/Backup". Select the "To String" option for the appropriate export type (icon, group, or profile), and then press CTRL+C to copy it to your clipboard.

Additionally, if applicable, add screenshots to help explain your problem. You can paste images directly into GitHub issues, or you can upload files as well.

**Comment by anianiani123** [comment]
^1^T^SGUID^STMW:group:1XcrZvoX84OO ^SScale^F4774481604116480 ^f-51^SRows ^N3^SPoint ^T^Sy^F-6853584230184499 ^f-46^Sx ^F-8724374278241359^f-50 ^Spoint^STOP ^SrelativePoint^STOP ^t^SEnabledSpecs^T ^N103^b ^N104^b ^N105^b ^t^SIcons^T ^N1^T ^SBuffOrDebuff^SHARMFUL ^SUnit^Starget ^SType^Scooldown ^SName^SEnvenom ^SEnabled^B ^SGUID^STMW:icon:1XcrZvaY=7S7 ^SConditions^T ^N1^T ^SType^SBUFFSTACKS ^SLevel^N2 ^SName^SEchoing~`Reprimand ^t^N2^T ^SType^SCOMBO ^SLevel^N2 ^t^Sn^N2 ^t^SStates^T ^N2^T ^SAlpha^N1 ^t^t^t^N2^T ^SBuffOrDebuff^SHARMFUL ^SUnit^Starget ^SType^Scooldown ^SName^SEnvenom ^SEnabled^B ^SGUID^STMW:icon:1XcrZvaesty0 ^SConditions^T ^N1^T ^SType^SBUFFSTACKS ^SLevel^N3 ^SName^SEchoing~`Reprimand ^t^N2^T ^SType^SCOMBO ^SLevel^N3 ^t^Sn^N2 ^t^SStates^T ^N2^T ^SAlpha^N1 ^t^t^t^N3^T ^SBuffOrDebuff^SHARMFUL ^SUnit^Starget ^SType^Scooldown ^SName^SEnvenom ^SEnabled^B ^SGUID^STMW:icon:1XcrZval1PyP ^SConditions^T ^N1^T ^SType^SBUFFSTACKS ^SLevel^N4 ^SName^SEchoing~`Reprimand ^t^N2^T ^SType^SCOMBO ^SLevel^N4 ^t^Sn^N2 ^t^SStates^T ^N2^T ^SAlpha^N1 ^t^t^t^N4^T ^SBuffOrDebuff^SHARMFUL ^SUnit^Starget ^SType^Scooldown ^SName^SEnvenom ^SEnabled^B ^SGUID^STMW:icon:1XcrZvar=2Cz ^SConditions^T ^N1^T ^SType^SBUFFSTACKS ^SLevel^N5 ^SName^SEchoing~`Reprimand ^t^N2^T ^SType^SCOMBO ^SLevel^N5 ^t^Sn^N2 ^t^SStates^T ^N2^T ^SAlpha^N1 ^t^t^t^N10^T ^SType^Smeta ^SIcons^T ^N1^STMW:icon:1XcrZvaY=7S7 ^N2^STMW:icon:1XcrZvaesty0 ^N3^STMW:icon:1XcrZval1PyP ^N4^STMW:icon:1XcrZvar=2Cz ^t^SEnabled^B ^t^t^t^N91100^S~`~| ^Sgroup^N79 ^^

**Comment by anianiani123** [comment]
!WA:2!Dr1tZTnru86jWaOEGe1shkduYKzOZuoKHMcLt9quQCtaxBZkfsyGzSxj9K1sK3vS7k7KCR(up7pc(mN8hHCGZA8W0pa5JqU1B82v2jT4dwV937TV)(7TnA6o0nXn51BnJfl4bIszm8L3sEAN0ufOBipBPWCkpoti7kyCD0E(Td9jlIfI8eXyEWywb0)Ivh9tgazEg31FIwYgmaKQh8q5sX34PpRai0sjDNPLkiqtJpr1FQbGthcQVOydFmum(GnjqHKnKYtiWiaJ7(anxN5vYz6OIC6zG8sL96DkajvlKB9SNfvJ4UZevbKNFqIYHy9RZLQYiRF6kHu2PKGU(TADnyqzkcoV3E7ge2liCxsOeZU24n7plbIqTHyEl33Vv3Mh2AAjFz948WkN)P62vUfUO99OrkH0MOVuKa9jfIXG0DNlTF7jUjpV8gRTwwToPRXMk3Q7m1ATPp9jMwup7X(tmIMAV)IRbn20V6RQEq1x7uSbnwZgbH1jMXR)9TUcNxGKtZ)vCkWe8F)QtaOyxS3eRjuntKnvb5P25A1hIvQIQXjHgYMHDgcmaVJ6BF9DlC1WP6EQmkoIpEjPO4(wq7FPc5qkAWsbpUGdtVw9wFJQy93Xd7jYfYFQrJgRDzn6QCWB3dd7uChliTulqxXIpILGJ9DXJxvhl2PqIf8vlQ5L)2Ym6cR()SuPzPNH1JuOXAjeb9A3PTF1TLyvzSzM1WyBASg(B6YIGRlwVPKD(M)sjnb7d0nddFVAFvKwyXglKjhjPftoAPqDcmIPyr5q)L187S7uSXbTB7t651jmSZljh8I9dxCDOdyNd3FzfD87fhJwl)RZHHToOT)BwmixmUPe(RsGhFw3p472ENFOU6mkMB(BLhMyoyU8IOsTwW7mcK4YZmdClGpqN9zw7xvztnh276oZfiz7fRqY8mkZSrpmJfFchuQ1S3iiMMd1ITyi8g2a4H9fqMfLbSbz6x5LlOjnJ0OPC99idlZ1mhpdB8Ev)HdjoNQugjpf2kmcZKwkOj9R(yPHGJ8XNsWLCP2jkLXzQmhp8J2zkLZm0fb)PvrnVaOMhxK2c8JWf0hViXqSr19mBnsfGVqLOMySZ6(h7nKY4nRsTwxnOkRIHFJ))iZz8Aoo6khVZfIHniJTKXzAr8O69SpVrV3UflP6tj3WXMNkXNtWCcPvRDv9yUPbYe(OG9i((Tj08Im6AryULYg4izCJdHSDkzjZhD8ZB(9YUp6rh)Zf3D1ZTgY9ZzkkY3sYMzQjMTj5mzLfzlWnjmwDkQv4Qipz7FC7N4o6Fp()(d

**Comment by anianiani123** [comment]
I must have missed the issue template, sorry.

The 2nd string is the weakaura. I just created an icon with two triggers, and it works like I want tellmewhen to work; the icon appears at two combo points.

**Comment by ascott18** [comment]
Thanks. It looks like this is working as intended. If you want to distinguish between multiple buffs with the same name, you need to track them by spellID.

You will also find helpful discussions on this topic in the #rogue channel in the TMW discord.

**Comment by anianiani123** [comment]
It works. Instead of "Echoing Reprimand", I am tracking the spell ID now.

Sorry that I wasted your time, but I really thought it was a bug, because "my method" worked on a weakaura icon.

I will also check discord in the future.

!!!Thank you for helping so quickly, ascott18!!!
